vtex-apps/store
# Add custom title and metatags to routes declaring it

#### What is the purpose of this pull request?
 Add custom title and metatags to routes declaring it.

#### What problem is this solving?
The user can't set description, keywords and title to custom pages.

#### How should this be manually tested?
Link this app in some `storecomponents` workspace, go to `/renato` then see that `title`, `description` and `keywords` are there.

#### Screenshots or example usage

Before:
<img width="1637" alt="Screen Shot 2019-05-24 at 16 51 08" src="https://user-images.githubusercontent.com/2726345/58353179-31ce7280-7e44-11e9-8be0-41bb48d74c9d.png">

After:
<img width="1607" alt="Screen Shot 2019-05-24 at 16 50 48" src="https://user-images.githubusercontent.com/2726345/58353198-372bbd00-7e44-11e9-8e57-6ef9ffdc4f85.png">

#### Types of changes
